examples/camera-app/linux/src/clusters/webrtc_provider/webrtc-provider-manager.cpp

```diff
@@ -226,7 +278,12 @@ void WebRTCProviderManager::ScheduleAnswerSend()
         CASESessionManager * caseSessionMgr = Server::GetInstance().GetCASESessionManager();
         VerifyOrDie(caseSessionMgr != nullptr);
 
-        caseSessionMgr->FindOrEstablishSession(mPeerId, &mOnConnectedCallback, &mOnConnectionFailureCallback);
+        // WebRTC Answer requires a large payload session establishment.
+        caseSessionMgr->FindOrEstablishSession(mPeerId, &mOnConnectedCallback, &mOnConnectionFailureCallback,
+#if CHIP_DEVICE_CONFIG_ENABLE_AUTOMATIC_CASE_RETRIES
+                                               1, nullptr,
+#endif // CHIP_DEVICE_CONFIG_ENABLE_AUTOMATIC_CASE_RETRIES
```

Comment:
See https://github.com/project-chip/connectedhomeip/pull/38370#discussion_r2043005056 -- the API being called here is obviously broken.  Followup to fix is fine but please do make sure it gets fixed.

Reply:
It is addressed by https://github.com/project-chip/connectedhomeip/pull/38370, will merge after #38370 landed